src/stack.js

```diff
@@ -0,0 +1,26 @@
+export default class Stack {
+  constructor() {
+    this.size = 0
+    this.elements = []
+  }
+
+  push = (elementToAdd) => {
+    this.elementToAdd = elementToAdd
```

Comment:
I think you are adding the elementToAdd as a property to the stack itself AND keeping track of the elementToAdd in the elements array. Seems a little redundant.